src/traits.rs

```diff
@@ -1,43 +1,9 @@
 //! Trait definitions
 use crate::data::DataOrSuffStat;
+use crate::suffstat_traits::*;
```

Comment:
Please mark as pub. They are currently private.

Reply:
ok changed to 
```rust
pub use crate::data::DataOrSuffStat;
pub use crate::suffstat_traits::*;
```